examples/r2i/tensorflow/meson.build

```diff
@@ -0,0 +1,13 @@
+# Compile examples
+app_examples = [
+  'inception',
+]
+
+common_inc_dir = '../common/'
```

Comment:
You could add this _common_inc_dir_ in the main meson.build file, because it seems to be used in multiple meson files.